app/controllers/concerns/backable.rb

```diff
@@ -18,8 +28,14 @@ def back_path
     private
 
     def update_page_history
+      return if self.class.skip_back_history_actions.include?(action_name.to_sym)
       return unless request.request_method_symbol == :get
 
+      process_history
+      remove_old_history
+    end
+
+    def process_history
```

Comment:
Split out this process logic because rubocop was complaining about the length of the `update_page_history` method